.github/workflows/test.yml

```
@@ -170,15 +155,12 @@ jobs:
 
       - name: Set up Xcode version
         run: sudo xcode-select -s /Applications/Xcode_16.1.app/Contents/Developer
-      
-      - run: bun install
-        working-directory: example
-      
+    
       - run: pod install
-        working-directory: example/ios
+        working-directory: apps/example/ios
       
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)